lib/Models/exercise.dart

```diff
@@ -3,18 +3,26 @@ import 'package:exerlog/Models/sets.dart';
 import 'package:exerlog/UI/exercise/totals_widget.dart';
 
 class Exercise {
+
+  Exercise(this.name, this.sets, this.bodyParts, this.totalSets, this.totalReps,
+      this.totalWeight);
```

Comment:
Add , for formatting

Reply:
ok i can fix this but remember that PR not fixes require_trailing_commas rule :P

Reply:
Didn't even know there was a rule for this haha. It just preferred to put a comma at the end to keep the formatting uniform.